src/site/content/en/blog/script-component/index.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: Optimizing third-party script loading in Next.js
+subhead: Understand the vision behind the Next.js’ Script component, which provides a built-in solution to optimize the loading of third-party scripts.
+date: 2022-02-23
+updated: 2021-02-23
+authors:
+  - leenasohoni
+  - houssein
+description: |
+  This article discusses the Next.js Script component and demonstrates how it can be used for better sequencing of third-party scripts.
+hero: image/IypihH3o5cSpEMVp5i08dp69otp2/sNDY4XVK4gyGHNKfdyJB.jpeg
+alt: A child arranging Jenga blocks
+tags:
+  - blog
+---
+
+Around [45%](https://almanac.httparchive.org/en/2021/third-parties#fig-3) of requests from websites served on mobile and desktop are third-party requests of which [33% are scripts](https://almanac.httparchive.org/en/2021/third-parties#fig-9). The size, latency, and loading of third-party scripts can significantly affect a site's performance. The Next.js Script component comes with baked-in best practices and defaults to help developers introduce third-party scripts in their applications while addressing potential performance issues out of the box. 
+
+
+## Third-party scripts and their impact on performance
+
+[Third-party scripts](/third-party-javascript/) allow web developers to leverage existing solutions to implement common features and reduce development time. But the creators of these scripts typically do not have any incentive to consider the performance impact on the consuming website. These scripts are also a blackbox to developers who use them. 
+
+Scripts account for a significant number of [third-party bytes](https://almanac.httparchive.org/en/2021/third-parties#fig-11) downloaded by websites across different categories of third-party requests. By default, the browser prioritizes scripts based on where they are in the document which may delay the discovery or execution of scripts critical to user experience. 
+
+Third-party libraries required for layout should be loaded early to render the page. Third-parties that are not required for initial render should be deferred so that they do not block other processing on the main thread. [Lighthouse](https://developers.google.com/web/tools/lighthouse) has two audits to flag render-blocking or main thread blocking scripts. 
+
+{% Img src="image/IypihH3o5cSpEMVp5i08dp69otp2/nzgIO68cBW9y5xCsW0g6.png", alt="Lighthouse audits for Eliminate render-blocking resources and Minimize third-party usage", width="800", height="301" %}
+
+It's important to consider the resource loading sequence of your page so that critical resources are not delayed and non-critical resources do not block the critical resources.
```

Comment:
```suggestion
It's important to consider the resource loading sequence of your page so that critical resources are not delayed and non-critical resources do not block critical resources.
```